lib/layout/raidlist.css

```diff
@@ -71,6 +71,35 @@
     background-image: url(images/raid_locked.png);
 }
 
+#raidlist .raidSlot .overlayStatus .ui-icon {
+    display: inline-block;
+}
+
+#raidlist .raidSlot .overlayStatus:before {
+    width: 28px;
+    height: 28px;
+    display: inline-block;
+    content: '';
+    margin-right: 16px;
```

Comment:
This is probably not a good idea as the position of the new element depends on the size of the element to it's right which is - afair - a number that can be either 1 or 2 digits wide.
In addition to that I would use a separate element, not before. This way you will see in the script that there is an element and don't have to look it up in CSS first.